index.html

```diff
@@ -356,12 +356,65 @@ <h2>
       </p>
       <aside class="issue" data-number="68"></aside>
     </section>
-    <section class="informative">
-      <h2>
-        Accessibility considerations
-      </h2>
-      <div class="issue" data-number="24"></div>
-    </section>
+    <h2>
```

Comment:
do we want an id here too, for consistency and linking?

Reply:
Actually, I can kill "displaying-a-badge" as I no longer link to it.